### PR TITLE
Retrieve hostname with multiple tags

### PR DESCRIPTION
**Pull request by k-chu**
This PR fixes an issue to retrieve hostname when there are multiple AWS tags.

### CODE REVIEW

**Comment by k-chu** [comment]
@miq-bot add_label kasparov/yes?

File: spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb [hunk 1]
```diff
@@ -89,6 +90,15 @@ def assert_specific_aws_configured_system(configuration_profile_id, orchestratio
       expect(aws_configured_system.counterpart).to eq(cross_link_aws_vm)
     end
 
+    def assert_aws_configured_system_hostname_with_multiple_tags
+      aws_configured_system = ems.configured_systems.find_by(:manager_ref => "5fb677b9ae457000181dd463")
+      expect(aws_configured_system).to have_attributes(
+        :type                     => "ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystem",
+        :hostname                 => "agostino-hybrid-1",
```
Comment (agrare) [inline spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb:97]: You shouldn't need this much leading whitespace, just enough to line it up with `:hostname =>`

Comment (k-chu) [reply spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb:97]: thanks, will fix it

Comment (k-chu) [reply spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb]: I pushed a fix. Thanks for the review!

**Review by agrare — APPROVED** [review]

**Comment by simaishi** [comment]
Kasparov backport details:

```
$ git log -1
commit 767e5116e476a1ea3ca8ec2842032c6d17a1bc2a
Author: Adam Grare <agrare@redhat.com>
Date:   Thu Nov 19 14:03:50 2020 -0500

    Merge pull request #50 from k-chu/retrieve_hostname

    Retrieve hostname with multiple tags

    (cherry picked from commit 7c04f7d4946dec6ea64bfc559f6de0e50413e345)
```

**Pull request merged**